components/button/button.css

```diff
@@ -0,0 +1,114 @@
+/* || ROOT CSS */
+:root {
+  --primary-bg-color: #60a5fa;
+  --primary-bg-hover-color: #4294f8;
+  --outline-hover-color: white;
+  --outline-basic-border-color: #78716c;
+  --outline-info-border-color: #fb923c;
+  --outline-danger-border-color: #dc2626;
+  --group-btn-bg-color: #d3d3d3;
+  --group-btn-active-color: black;
+  --float-btn-shadow-color: #00000066;
+  --btn-link-color: #3066db;
+}
+
+/* || PRIMARY BUTTON */
+.btn {
+  margin: 0.5rem;
+  background-color: transparent;
+  outline: transparent;
+  border-radius: 0.3rem;
+  border: 0;
+  text-align: center;
+  cursor: pointer;
+}
+.btn-sm {
+  padding: 0.5rem 1rem;
+}
+.btn-md {
+  padding: 0.5rem 2rem;
+}
+.btn-lg {
+  padding: 0.8rem 3.5rem;
+}
```

Comment:
use Pixel for less than 1rem.

Reply:
I've done this in too many places to change now. 😬